build/build.ps1

```diff
@@ -7,8 +7,8 @@ $ErrorActionPreference = 'Stop'
 $all_ok = $True
 
 if ($Env:ENABLE_QIRRUNTIME -ne "false") {
-    $qirRuntime = (Join-Path $PSScriptRoot "../src/Qir/Runtime")
-    & "$qirRuntime/build-qir-runtime.ps1"
+    $qirstdlib = (Join-Path $PSScriptRoot "../src/Qir/Runtime")
+    & "$qirstdlib/build-qir-stdlib.ps1"
```

Comment:
(Feel free to ignore)
For the variables that contain the paths I prefer the name of the var to end with "Path" (and sometimes it really helps if the name ends with either "FilePath" or "DirPath").

Reply:
Updated to include Path in the name!